solr/bin/solr

```diff
@@ -1581,7 +1581,8 @@ if [ "${SOLR_HOME:0:${#EXAMPLE_DIR}}" = "$EXAMPLE_DIR" ]; then
   SOLR_LOGS_DIR="$SOLR_HOME/../logs"
 fi
 
-LOG4J_CONFIG=()
+# Set the logging manager by default, so that Lucene JUL logs are included with Solr logs.
+LOG4J_CONFIG=("-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager")
```

Comment:
It looks like this is misisng for Windows `solr.cmd`.

Small question: I hope this is an internal variable that is not overwritten by user code in `solr.in.sh`?

Reply:
No, `solr.in.sh` cannot override this. But good call on windows, will add that in.

Reply:
Ok it should work for windows too, but it does it in a slightly different way. Shouldn't be a big deal.